app/controllers/staff/organisations_controller.rb

```diff
@@ -25,10 +25,13 @@ def show
         @third_party_project_activities = third_party_project_activities.map { |activity| ActivityPresenter.new(activity) }
       end
       format.xml do
-        @project_activities = if third_party_project_activities.present?
+        @activities = case level
+        when "c"
```

Comment:
I think it would be preferable to keep using consistent language internally for `projects` and `third-party projects` rather than adding a mix of both for simplicity sake. I suppose it would make the query param longer but I think that's okay?

Reply:
Good call!